src/main/interface/settings.c

```diff
@@ -682,6 +682,7 @@ const clivalue_t valueTable[] = {
     { "use_cbat_alerts",            VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, useConsumptionAlerts) },
     { "cbat_alert_percent",         VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 100 }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, consumptionWarningPercentage) },
     { "vbat_cutoff_percent",           VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 100 }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, lvcPercentage) },
+    { "set_battery_cell_count",     VAR_UINT8  | MASTER_VALUE, .config.minmax = { 10, 80 }, PG_BATTERY_CONFIG, offsetof(batteryConfig_t, setBatteryCellCount) },
```

Comment:
minmax = { 10, 80 } ?

Reply:
What I though it would mean is that 10 means 1 and 80 8, so a value between 1 and 8 should be accepted. Thinking about it, 0 should be allowed too.

Reply:
What I though it would mean is that 10 means 1 and 80 8, so a value between 1 and 8 should be accepted. Thinking about it, 0 should be allowed too.

If I'm correct in the above, in the code 8s is considered the maximum possible value you're allowed to set, so I wanted to keep in line with the current setting and 80 should be correct.